core/templates/components/certificate-assessment-offering-helper/assessment-interrupt.component.html

```
@@ -0,0 +1,27 @@
+<div class="assessment-interrupt-card">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)